src/coreclr/nativeaot/Runtime/eventpipe/ep-rt-types-aot.h

```diff
@@ -27,6 +27,8 @@
 #define EP_ASSERT(expr)
 #endif
 
+#define EP_FAILFAST(msg) 
```

Comment:
It wasn't clear that we really needed a failfast based on how it was being used, but if we do then this needs a real implementation right?